### PR TITLE
fix: update code and sequelize version to avoid crash

### PR DESCRIPTION
**Pull request by arnaud-moncel**
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)
- [ ] Ensure that Types have been updated according to your changes (if needed)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: packages/datasource-sequelize/src/utils/query-converter.ts [hunk 1]
```diff
@@ -44,13 +42,14 @@ export default class QueryConverter {
     operator: Operator,
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     value?: any,
-  ): WhereOperators | OrOperator | Where {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ): any {
```
Comment (jeffladiray) [inline packages/datasource-sequelize/src/utils/query-converter.ts:46]: This is weird though, I'll look if Sequelize provide a better type than `any`

**Review by jeffladiray — COMMENTED** [review]
I still need to check if everything works as expected for older sequelize version (>6)

File: packages/datasource-live/package.json [hunk 2]
```diff
@@ -29,9 +29,9 @@
   },
   "devDependencies": {
     "lodash": "^4.17.21",
-    "sequelize": "6.18.0"
+    "sequelize": "6.21.3"
   },
   "peerDependencies": {
```
[the remark this answers is not in the document]
Comment (romain-gilliotte) [reply packages/datasource-live/package.json:34]: For the live datasource and sql datasource it should actually not be a peerdep at all but a simple dependency

Customers don't need to know that we use sequelize (and should be able to use another incompatible version in the same project without interactions)

File: packages/datasource-sequelize/package.json [hunk 3]
```diff
@@ -27,10 +27,10 @@
   },
   "devDependencies": {
     "fishery": "^2.1.0",
-    "sequelize": "6.18.0",
+    "sequelize": "6.21.3",
     "sqlite3": "^5.0.2"
   },
   "peerDependencies": {
-    "sequelize": "6.18.0"
+    "sequelize": "6.21.3"
```
Comment (romain-gilliotte) [inline packages/datasource-sequelize/package.json:34]: This should be 6.x.x

Customers will have warning if they don't use this exact version

**Review by romain-gilliotte — DISMISSED** [review]

**Review by jeffladiray — COMMENTED** [review]
As we discover, using anything regarding sequelize should use the Sequelize instance provided by the user.
I'm pretty sure this is worth mentioning somewhere for futur developers as a comment, since it was a real pain to understand that in the first place.

(The fix looks good to me, but I'll do a few other tests before approving)

**Review by jeffladiray — APPROVED** [review]

**Pull request merged**